huggingface_datasets_converter/convert.py

```diff
@@ -96,6 +96,14 @@ def get_kaggle_metadata(kaggle_id):
     except:
         license = 'unknown'
 
+    if license == 'unknown':
```

Comment:
I think if license is other we should also raise the error

Reply:
You are right! I forgot that part 😅